stl/inc/chrono

```diff
@@ -5351,6 +5418,24 @@ namespace chrono {
 
                     _Validate_specifiers(_Spec, _Val);
 
+#if defined(_MSVC_EXECUTION_CHARACTER_SET) && _MSVC_EXECUTION_CHARACTER_SET == 65001 // TRANSITION, VSO-1468747 (EDG)
+                    if constexpr (is_same_v<_CharT, char>) {
+                        if (_Specs._Localized) {
+                            wostringstream _Wstream;

```

Comment:
Can we use `_wcsftime_l` instead? I might just hate iostreams, but I would personally prefer to use the C standard library directly here...

Reply:
it's to match what the formatter was doing below, and to match how chrono formatting is specified.